CMakeModules/nsis/NSIS.template.in

```diff
@@ -815,7 +815,7 @@ SectionEnd
 ;--------------------------------
 ; Component dependencies
 Function .onSelChange
-  !insertmacro SectionList MaybeSelectionChanged
+  !insertmacro SectionList "MaybeSelectionChanged"
```

Comment:
we haven't changed NSIS template for a while and they haven't given issues. i hope you are testing NSIS changes

Reply:
Yes of course. I found this because there was an error that popped up during the making of the installer.

Reply:
just being cautious of changing NSIS :)